### PR TITLE
Update docker to use celery via redis

### PR DESCRIPTION
**Pull request by robhudson**
Github issue: https://github.com/mozilla/marketplace-env/issues/95

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,5 +42,10 @@ ENV SOLITUDE_URL http://solitude:2602
 ENV STYLUS_BIN /srv/zamboni-node/node_modules/stylus/bin/stylus
 ENV UGLIFY_BIN /srv/zamboni-node/node_modules/uglify-js/bin/uglifyjs
 ENV ZAMBONI_DATABASE mysql://root:@mysql:3306/zamboni
+ENV CELERY_BROKER_URL redis://redis:6379/1
+ENV CELERY_RESULT_BAKEND redis://redis:6379/2
```
Comment (andymckay) [inline Dockerfile]: should that be BACKEND?

Comment (robhudson) [inline Dockerfile]: heh, yeah. At least I was consistent!

File: scripts/docker/supervisor.conf [hunk 2]
```diff
@@ -18,3 +18,12 @@ port=9001
 
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:celeryd]
+command=python manage.py celery worker --autoreload --broker=redis://redis:6379/1 -Q priority,devhub,images,limited,celery --loglevel DEBUG
```
Comment (robhudson) [inline scripts/docker/supervisor.conf]: I'll remove the loglevel here, it was for testing. Although it *is* nice for logging. Hmm.

**Comment by andymckay** [comment]
The forcing of JSON solves the problems with users? r+

**Comment by robhudson** [comment]
It forces celery to allow being root and using pickle. So that avoids having to create a user and deal with file permissions yeah. My next goal is to get our tasks to not pass objects and then switch to json which is more secure and then we can drop this.

**Comment by andymckay** [comment]
Sounds great.

File: scripts/docker/supervisor.conf (hunk 2 above)
Comment (robhudson) [inline scripts/docker/supervisor.conf]: Yeah, I decided to leave this.

**Comment by robhudson** [comment]
Thanks @andymckay

**Pull request merged**